JustSaying/JustSayingBus.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace JustSaying
 {
-    public class JustSayingBus : IAmJustSaying, IAmJustInterrogating
+    public sealed class JustSayingBus : IAmJustSaying, IAmJustInterrogating
     {
         public bool Listening { get; private set; }

```

Comment:
Do we still need this property (I think it's edge case, but yet). If so, we need to set it to false when cancelling.

Reply:
It would be easier to delete it and have whoever consumes this keep track of the listen status as they will own the cancellation token

Reply:
Posible solution:
```csharp
public void Start(CancellationToken cancellationToken = default)
{
    cancellationToken.Register(() => Listening = false);
    ...
```

It might be better to handle it in a more coordinated way with the task that launches the listen loop though.

Reply:
I've just gone through some scenarios in my head, and all of them convince me that we should get rid of it as you suggest 👍